cmd/cloud-controller-manager/app/apis/config/v1alpha1/defaults.go

```diff
@@ -29,7 +29,9 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_CloudControllerManagerConfiguration(obj *CloudControllerManagerConfiguration) {
+// SetDefaultsCloudControllerManagerConfiguration is used to set default values for
+// some CloudControllerManagerConfiguration's fields
+func SetDefaultsCloudControllerManagerConfiguration(obj *CloudControllerManagerConfiguration) {
```

Comment:
this is related to https://github.com/kubernetes/code-generator/issues/30
i don't think the underscores can be fixed before the above issue.